src/components/ProductCard.jsx

```diff
@@ -1,19 +1,13 @@
-export const ProductCard = ({ props }) => {
```

Comment:
UI suggestion: add some spacing inside the card 
![image](https://user-images.githubusercontent.com/67889207/159264175-47f1920b-33ca-4f69-beb3-86ed1ab6a17b.png)

Reply:
Done